packages/terra-clinical-header/src/Header.scss

```diff
@@ -32,7 +32,7 @@
   }
 
   .flex-end {
-    align-self: center;
+    align-items: center;
```

Comment:
I believe this is the right change to fix the issue, but it may have a breaking impact on one of our internal themes.

cc: @yuderekyu

Reply:
I made this change to try and fix an alignment issue in the clinical theme. In hindsight it was a regression sorry :(

Can we open a verification branch in the clinical theme for this?

Reply:
@yuderekyu I have created a branch for verification.
https://github.cerner.com/terra/cerner-clinical-theme/tree/verify-clinical-theme-clinical-header-alignment.

Also I tried testing this on Jenkins. Was unable to run the tests due to limited resource.
https://github.cerner.com/terra/cerner-clinical-theme/tree/verify-clinical-theme-clinical-header-alignment.

Have a look and let me know your thoughts. Thank you.

Reply:
The verification branch is pointed to a master snapshot of clinical header - we need to generate a snapshot of clinical header pointed to `terra-clinical-header-alignment` and update the package.json within clinical theme to point to this generated snapshot

Reply:
My bad, shared the branch without committing the changes. I have updated the branch. 
Link -> https://github.cerner.com/terra/cerner-clinical-theme/tree/verify-clinical-theme-clinical-header-alignment

Reply:
sorry for the back and forth - can you create a pull request there? branch builds don't actually run tests, we need a pull request to trigger them

Reply:
Here is a [Pull Request](https://github.cerner.com/terra/cerner-clinical-theme/pull/232) for the same.

Reply:
> Here is a [Pull Request](https://github.cerner.com/terra/cerner-clinical-theme/pull/232) for the same.

Updated Screenshots for clinical-header in terra-clinical-theme since tests were failing.

Reply:
@lokesh-0813 make sure to update the screenshots for other theming repo's as well. once this is merged.